storybooknative/src/styled/__tests__/index.test.js

```diff
@@ -0,0 +1,109 @@
+// @flow
+/* eslint-disable no-return-assign */
+import { Platform } from "react-native";
+
+import { createPlatformStyles } from "../index";
+
+let originalPlatform;
+// $FlowFixMe
+beforeEach(() => (originalPlatform = Platform.OS));
+// $FlowFixMe
+afterEach(() => (Platform.OS = originalPlatform));
+
+const mockStyles = {
+  fontSize: 20,
+  fontWeight: "bold",
+  ios: {
+    backgroundColor: "blue",
+  },
+  android: {
+    backgroundColor: "green",
+  },
+  web: {
+    backgroundColor: "orange",
+  },
+};
+const fontSize = 10;
```

Comment:
I am not sure about the `fontSize`'s being numbers. I think for it to work with react-native-web, we need them to be in `px`, like `"10px"`.

Reply:
good point, thanks